### ISSUE TITLE
Game's ending doesn't bring back to main screen

### ISSUE THREAD

**Issue by Mastergatto** [sm64pc/sm64pc#115]
Since the game gets 'stuck' at the ending screen as it's intended by Nintendo developers back in the N64 era, and if you wanted to still play it you had to press the reset button.

Now, on PC you can't do that, but what if instead the game brings you automatically to its main screen/intro. That would be a neat thing.

**Comment by Leon422** [comment]
Agreed. The game getting "stuck" in the ending screen makes no sense in a modern platform, and since no one does it anymore, it gives the impression that the game is hung.

**Comment by GateGuy** [comment]
#283
